main.py

```diff
@@ -1,7 +1,55 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    # Create genres
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the code, but it's better to use the correct name initially to avoid unnecessary updates.